travel_platform/skyscanner_data/skyscanner_data_tests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from skyscanner_data import SkyscannerData
+
+# FIXME: which parameters there are must be expected? FAILED (errors=4)
```

Comment:
Well, as I mentioned previously in `FIXME` section we don't expect some question/thoughts about life, etc.

It's just an informative block.

So, it should looks similar to:

```
# FIXME Sasha: Fix all these tests, they fails because of invalid parameters.
```